### ISSUE TITLE
[FR]: `/p unmerge` command

### ISSUE THREAD

**Issue by supercrafter333** [ColinHDev/CPlot#5]
Hello,
Can you add a command for unmerging plots, please?

:)

**Comment by ColinHDev** [comment]
This existed in a former version but it basically just unmerged all plots without clearing the plots themselves or removing the owner. But I didn't like that approach as that meant that a merged plot with the size of for example 20 plots would completely be split up into separate ones, while you maybe would only want to remove the connection between two plots.
So please explain further what that command should actually do in your opinion.

**Comment by supercrafter333** [comment]
Unmerge should unmerge the opposide plot without removing the owner.

**Comment by ColinHDev** [comment]
Yes, I know. But there isn't always just a simple plot that gets unmerged.
Merged Plots can have many different shapes and that makes it really difficult to implement.
Look at the example below
![IMG_20220114_210313.jpg](https://user-images.githubusercontent.com/54852588/149577879-e7a0ae24-b7a2-4e39-beb4-09b59443f715.jpg)

**Comment by supercrafter333** [comment]
Okay...

**Comment by supercrafter333** [comment]
Please do this before myplot intigration. MyPlot is shit :))))))

**Comment by ColinHDev** [comment]
MyPlot-Support will be the final update before 1.0.0 and before I wanted to work on `/p unmerge` I wanted to finish https://github.com/ColinHDev/CPlot/commit/4487965645c09493989210d97ade13ec946dab08.
So this will be addressed as one of the next things.

**Comment by supercrafter333** [comment]
Okay. Thank you :)

**Comment by supercrafter333** [comment]
Could be a feature for CPlot PM5.

**Comment by ColinHDev** [comment]
There is not really a difficulty in implementing this for the normal `development` branch. The PM5 milestone is not really an update, only a bunch of things that need to be addressed regarding PM5.
I just don't think that this feature is important enough to get done before the 1.0.0 update.
